plugins/dfu/fu-dfu-device.c

```diff
@@ -1971,7 +1948,4 @@ fu_dfu_device_init(FuDfuDevice *self)
 					FU_DFU_DEVICE_FLAG_NO_BUS_RESET_ATTACH,
 					"no-bus-reset-attach");
 	fu_device_register_private_flag(FU_DEVICE(self), FU_DFU_DEVICE_FLAG_GD32, "gd32");
-	fu_device_register_private_flag(FU_DEVICE(self),
```

Comment:
Sorry for the late comment. Is there any reason to remove this custom flag? It was introduced in https://github.com/fwupd/fwupd/pull/3641/commits/442cd0db4cc7a663aa871756d2ed9ac2009c99b0 for fixing a slow downloading issue with some Poly devices.

Reply:
Removing this was totally unintentional! I'm so sorry! I'll add it back straight away.

Reply:
Okay, I've committed this back into main. I'll do a new upstream tarball this week to include this. I have no idea what happened here, mea culpa.

Reply:
Thanks!